syncd/Makefile.am

```diff
@@ -20,6 +20,12 @@ if SAICAVM
 SAIFLAGS = -DCAVMSAI
 endif
 
+if SAIVS
+SAILIB=-L$(top_srcdir)/vslib/src/.libs -lsaivs
```

Comment:
its needed since i want to prefer local build lib agains the one installed in the system which is causing a lot of confusion and require to build twice